### PR TITLE
Fixes Landscape configuration.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#390 solved. @deshraj Please review the PR.

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@deshraj For now, I have ignored the `docstrings` in all the files. When the issue #327 is solved, then please enable it from the `.landscape.yml` and `.pylintrc` files.

**Comment by TroJan** [comment]
@RishabhJain2018 : what if we break this pr per app, problem is we usually avoid so much code changes(this involves more than 15 files) in a single app.

Doing changes per app will make it neat, easy for us to review also

**Comment by RishabhJain2018** [comment]
@TroJan yeah sure, we can do that. Moreover, I would like to suggest that If within this PR only, I will remove all the present commits & push the per app basis commits. Will it do any good?

**Comment by dexter1691** [comment]
@TroJan: Thanks!!!
Can we please break these down into smaller more meaningful commits!

**Comment by RishabhJain2018** [comment]
@dexter1691 yeah sure. I am breaking down this PR into smaller per app basis commits.

**Comment by RishabhJain2018** [comment]
@dexter1691 @TroJan Please review the PR.

**Comment by deshraj** [comment]
@RishabhJain2018 can you please divide this PR into different PRs per app?

**Comment by deshraj** [comment]
Also, do update your fork before pushing.

**Comment by RishabhJain2018** [comment]
@deshraj yeah sure.

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,407 @@
+[MASTER]
```
Comment (deshraj) [inline .pylintrc]: @RishabhJain2018 is this file is really required here?

Comment (RishabhJain2018) [inline .pylintrc]: @deshraj yeah, This file is for the pylint configuration. Like if we have to disable a pylint warning on complete project then we can do it using `disable` option within this file. Moreover, it is to make pylint checks according to our requirements, like `max-line-length` [here](https://github.com/Cloud-CV/EvalAI/pull/398/files#diff-d6d3d73911fd3ad6f39e4027f763b307R291) is set to `120`, whereas the default is `80`. There are many other features also which can be enabled or disabled.

Comment (deshraj) [inline .pylintrc]: I think we are already using `flake8`. So, this would be just a duplicate thing to use for the same purpose.

Comment (RishabhJain2018) [inline .pylintrc]: @deshraj Please have a look at the comparison of `flake8` and `pylint` [here](https://python.libhunt.com/project/pylint/vs/flake8). Pylint is a good, handy, convenient and wonderful tool to be used.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[MESSAGES CONTROL]
+# Disable Messages
+disable=C1001,R0903,W0613,W0702,E1101,D100,C0330,W0223,C0111,W0232,C0103,R0201
```
Comment (deshraj) [inline .pylintrc]: @RishabhJain2018 can you add comment to show what all these mean?

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
```
Comment (deshraj) [inline .pylintrc]: Please remove this line.

File: .pylintrc (hunk 2 above)
Comment (RishabhJain2018) [inline .pylintrc]: yeah sure.

File: .pylintrc (hunk 3 above)
Comment (RishabhJain2018) [inline .pylintrc]: okk

**Comment by deshraj** [comment]
Looks good. Great work. :-)

**Pull request merged**